### PR TITLE
Added linting in few modules of the codebase

### PR DESCRIPTION
**Pull request by utkarshverma11**
Hi @samarth-p,

There are some linting fixes that I noticed while going through the codebase. Linting highlights the syntactical and and stylist problems which oftentimes helps to identify and correct subtle programming errors.

Details about the linter used
Used autopep8 python linter from within spyder editor

Fixed the following during the static code analysis:
Covention
Refractor
Warnings
Errors

Could you please review the fixes and let me know if they look fine then will go ahead and work on other modules that require linting.

Happy Linting ! :)

Thanks

### CODE REVIEW

File: info/admin.py [hunk 1]
```diff
@@ -5,7 +5,8 @@
 from django.http import HttpResponseRedirect
 from django.urls import path
 
-from .models import Dept, Class, Student, Attendance, Course, Teacher, Assign, AssignTime, AttendanceClass
+from .models import Dept, Class, Student, Attendance, Course, Teacher, Assign, 
+AssignTime, AttendanceClass
```
Comment (samarth-p) [inline info/admin.py:9]:
```suggestion
from .models import Dept, Class, Student, Attendance, Course, Teacher, Assign, \
    AssignTime, AttendanceClass
```

File: info/apps.py [hunk 2]
```diff
@@ -1,5 +1,4 @@
 from django.apps import AppConfig
 
-
```
Comment (samarth-p) [inline info/apps.py:3]: I think there needs to be 2 blank lines before class defintion according to PEP-8

**Review by samarth-p — CHANGES_REQUESTED** [review]

**Comment by samarth-p** [comment]
Hi @utkarshverma11 

I have gone through the changes. I have given a couple comments other than that everything looks fine.

Thanks a lot for making these changes!

File: info/admin.py (hunk 1 above)
Comment (utkarshverma11) [reply info/admin.py:9]: yep missed the "/" sign thanks for pointing out.

File: info/apps.py (hunk 2 above)
Comment (utkarshverma11) [reply info/apps.py:3]: Sure I will add the change now.

Comment (utkarshverma11) [reply info/apps.py]: Done. Please check

**Comment by utkarshverma11** [comment]
> Hi @utkarshverma11
> 
> I have gone through the changes. I have given a couple comments other than that everything looks fine.
> 
> Thanks a lot for making these changes!

**Pull request closed without merging**